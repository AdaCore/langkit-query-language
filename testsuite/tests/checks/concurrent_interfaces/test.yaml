driver: 'checker'
rule_name: concurrent_interfaces
input_sources: ['interfaces.ads']
