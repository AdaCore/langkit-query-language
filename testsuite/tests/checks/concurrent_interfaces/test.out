interfaces.ads:4:31: rule violation: concurrent interface
4 |    type Synchronized_Queue is synchronized interface and Queue;       --  FLAG
  |                               ^^^^^^^^^^^^

interfaces.ads:5:30: rule violation: concurrent interface
5 |    type Synchronized_Task is task interface;                          --  FLAG
  |                              ^^^^

interfaces.ads:6:35: rule violation: concurrent interface
6 |    type Synchronized_Protected is protected interface;                --  FLAG
  |                                   ^^^^^^^^^

